diff of execution of:
echo 'export PATH="$PATH:$PWD"; mkdir asdfg; asdfg' | tr ';' '\n' | [shell]
--- expected	2024-09-02 12:53:32.817833397 +0200
+++ found	2024-09-02 12:53:32.925832487 +0200
@@ -1,5 +1,6 @@
 
 ======== stderr =======
+[progname]: mkdir: command not found
 [progname]: asdfg: command not found
 ======== return =======
 return value: 1
